### ISSUE TITLE
Use aria-activedescendant?

### ISSUE THREAD

**Issue by sarbbottam** [sarbbottam/react-accessible-typeahead#1]
`aria-activedescendant` should be more useful than `aria-live` region. Please refer https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant.
